.github/workflows/installer-pr.yml

```diff
@@ -95,3 +98,59 @@ jobs:
             ```
             ${{ steps.read.outputs.content }}
             ```
+  create-harvester-pr:
+    runs-on: ubuntu-latest
+    needs: generate-image-diff
+    steps:
+      - uses: jungwinter/split@v2
+        id: spliter
+        with:
+          msg: ${{ github.ref_name }}
+          separator: '-'
+      - name: Download container-diff result
+        uses: actions/download-artifact@v3
+        with:
+          name: diff-result
+      - name: Read diff result
+        id: read
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./diff-result.txt
```

Comment:
The diff result is about the harvester-os image, do we need this in harvester/harvester PR?

Reply:
Nice catch... fixed!